docs/install.md

```diff
@@ -936,3 +936,189 @@ This procedure can be used to downgrade the active version of SAT.
+
+SAT can optionally be installed and configured on an external system to interact with CSM over the CAN.
+
+### Limitations
+
+Most SAT subcommands work by accessing APIs which are reachable via the CAN. However, certain SAT commands depend on
+host-based functionality on the management NCNs and will not work from an external system. This includes the following:
+
+- The `platform-services` and `ncn-power` stages of `sat bootsys`
+- The local host information displayed by the `--local` option of `sat showrev`
+
+Installing SAT on an external system is not an officially supported configuration. These instructions are provided
+"as-is" with the hope that they can useful for users who desire additional flexibility.
+
+Certain additional steps may need to be taken to install and configure SAT depending on the configuration of the
+external system in use. These additional steps may include provisioning virtual machines, installing packages, or
+configuring TLS certificates, and these steps are outside the scope of this documentation. This section covers only the
+steps needed to configure SAT to use externally-accessible API endpoints exposed by CSM.
+
+### Prerequisites
+
+- The external system must be on the Customer Access Network (CAN).
+- Python 3.7 or newer is installed on the system.
+- `kubectl`, `openssh`, `git`, and `curl` are installed on the external system.
+- The root CA certificates used when installing CSM have been added to the external system's trust store such that
+  authenticated TLS connections can be made to the CSM REST API gateway. See the [Certificate
+  Authority](https://cray-hpe.github.io/docs-csm/en-12/background/certificate_authority/) section of the CSM
+  documentation for more information.
+
+### Procedure
+
+1.  Create a Python virtual environment.
+
+    ```screen
+    $ SAT_VENV_PATH="$(pwd)/venv"
+    $ python3 -m venv ${SAT_VENV_PATH}
+    $ . ${SAT_VENV_PATH}/bin/activate
+    ```
+
+1.  Install the SAT Python package in the virtual environment.
+
+    ```screen
+    (venv) $ export PIP_EXTRA_INDEX_URL="https://artifactory.algol60.net/artifactory/csm-python-modules/simple"
+    (venv) $ git clone --branch=release/3.17 https://github.com/Cray-HPE/sat.git
+    (venv) $ pip install -r sat/requirements.lock.txt
+    ...
+    (venv) $ pip install ./sat
+    ...
+    ```
+
+1.  Optional: Add the `sat` virtualenv to the user's `PATH` environment variable.
+
+    If a shell other than `bash` is in use, replace `~/.bash_profile` with the appropriate profile path.
+
+    If the virtualenv is not added to the user's `PATH` environment variable, then
+    `source ${SAT_VENV_PATH}/bin/activate` will need to be run before running any SAT commands.
+
+    ```screen
+    (venv) $ deactivate
+    $ echo export PATH=\"${SAT_VENV_PATH}/bin:${PATH}\" >> ~/.bash_profile
+    $ source ~/.bash_profile
+    ```
+
+1.  Copy the file `/etc/kubernetes/admin.conf` from `ncn-m001` to `~/.kube/config` on the external system.
```

Comment:
I definitely agree that it would require configuration beyond what is there by default, though I would not be surprised if this is something sites regularly customize. I do feel like doing that configuration is outside the scope of this particular guide, and a site who has set up finer-grained access would probably also be able to set up the client authentication for the external host. I guess we could consider these to be like "reference instructions" that use the default configuration.

I'm having kind of a hard time deciding what to do, but I'm leaning towards just leaving it or at most cautioning that this will give the external host the same level of access to kubernetes as root on the NCNs has.

As an aside I feel like if a site doesn't want to share `admin.conf` and also doesn't know how to set up other user access, then they would just skip these steps and have a partially functional `sat`. But we haven't really tested the different ways in which the non-functional commands would fail.